CONTRIBUTING.md

```diff
@@ -6,43 +6,57 @@ happen through Git and Pull Requests.
 
 ## Requirements
 
-To contribute to ECS the following tools are expected to be running on your machine:
+You need these tools to contribute to ECS:
 
 * [Git](https://git-scm.com/)
 * [Python](https://www.python.org/)
 * [Go](https://golang.org/)
 
 ## Steps to contribute
 
-To contribute changes to ECS follow the steps below:
-
-* Create [your own fork](https://help.github.com/articles/fork-a-repo/) of the ECS repo.
-* Clone your fork to your machine.
-* Run `git checkout -b branch-name` and replace `branch-name` according to your change
-* Apply your changes to the `.yml` files as needed
-* Run `make` to update generated files like `schema.csv` and `schema.md`
-* Run `git commit -a -m "your message"`
-* Run `git push --set-upstream origin branch-name` (assumes your clone remote is called `origin`)
-* Create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) against the ECS repo. (if you go to the website, it should pop up as a button with your branch directly)
-* Wait for reviews on your PR and collaborate.
-
-Notes: Make sure to always only push changes against your own fork.
-
+Here are the steps for contributing to ECS.
+
+* Set up your git environment.
+  - Create [your own fork](https://help.github.com/articles/fork-a-repo/) of the ECS repo.
+  - Clone your fork to your machine.
+* Create a working branch on your machine.
+  - Run `git checkout -b branch-name`, where `branch-name` is the name you want to give your local branch
+* Do your work. 
+  - Make changes to the `.yml` files as needed
+* Run `make` to update generated files such as `schema.csv` and `schema.md`
+* Commit your changes locally. 
```

Comment:
I wonder if we really still need that? The reason I put it here is that I hope not only people with Git experience Contribute. Perhaps we can mention here instead that contributions can also be done through opening a Github issue?

Reply:
Git can be intimidating. Unless this is misleading or wrong, it's good to have it in there--especially if you want to invite new people to contribute.  Experienced users can scan the list and keep going.  

I added an "Other ways to contribute" section based on your comment about inviting contributors who are not experienced with Git.  Let me know what you think.